### PR TITLE
feat(docs): Add seldon core install docs

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @gaocegege @FogDong 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ $ make docker-build
 #### Install Harbor
 [Harbor](https://github.com/goharbor/harbor) is registry for the training model in klever-model-registry, please refer to the installation of Harbor [harbor-helm installation](https://github.com/goharbor/harbor-helm)
 
+#### Install Seldon Core
+
+Please have a look at [docs/installer/seldon-core/install.md](docs/installer/seldon-core/install.md)
```
Comment (FogDong) [inline README.md:87]: Should we need a new page of installation or we can just write it in this doc?

Comment (gaocegege) [reply README.md:87]: I have no opinion on it. WDYT @FogDong

Comment (simon-cj) [reply README.md]: Comments are addressed, PTAL @FogDong @gaocegege

**Pull request merged**